Assets/GothicVR/Scripts/Creator/Meshes/WorldMeshCreator.cs

```diff
@@ -24,6 +24,14 @@ public async Task<GameObject> CreateAsync(WorldData world, GameObject parent, in
             
             foreach (var subMesh in world.subMeshes.Values)
             {
+                // No texture to add.
+                // For G1 this is: material.name == [KEINE, KEINETEXTUREN, DEFAULT, BRETT2, BRETT1, SUMPFWAASER, S:PSIT01_ABODEN]
+                // Removing these removes tiny slices of walls on the ground. If anyone finds them, I owe them a beer.
```

Comment:
![image](https://github.com/GothicVRProject/GothicVR/assets/23665744/a62c99ae-4d18-4524-92b5-273fc7032e2b)
i can't wait for the beer

jokes aside, shouldn't we follow the example from opengothic to use DEFAULT.TGA if no texture could be found and if texture is empty then to use the color as a texture?

Reply:
Haha. Try again...
It's a bug in the original game as well. ;-)
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/025d8f30-9259-4063-9010-a0552cba33e1)

Reply:
@JucanAndreiDaniel any other aspects I shall redo or are you fine with the branch?